pkg/services/sqlstore/stats.go

```diff
@@ -183,3 +184,18 @@ func GetSystemUserCountStats(ctx context.Context, query *models.GetSystemUserCou
 		return err
 	})
 }
+
+func GetActiveUserCountStats(query *models.GetActiveUserCountStatsQuery) error {
+	activeEndDate := time.Now().Add(-activeUserTimeLimit)
+
+	var rawSql = `SELECT COUNT(*) AS Count FROM ` + dialect.Quote("user") + ` where last_seen_at > ?`
+	var stats models.ActiveUserCountStats
+	_, err := x.SQL(rawSql, activeEndDate).Get(&stats)
+	if err != nil {
+		return err
+	}
+
+	query.Result = &stats
+
+	return err
```

Comment:
At this point we know there is no error, so better be explicit:
```suggestion
	return nil
```